tests/e2e/statefulsets.go

```diff
@@ -82,18 +82,34 @@ var _ = ginkgo.Describe("statefulset", func() {
 		stsReplicas                int32
 		allowedTopologies          []v1.TopologySelectorLabelRequirement
 		isQuotaValidationSupported bool
+		adminClient                clientset.Interface
 	)
 
 	ginkgo.BeforeEach(func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
-		namespace = getNamespaceToRunTests(f)
+		var err error
 		client = f.ClientSet
+		namespace = getNamespaceToRunTests(f)
+		if supervisorCluster {
+			if svAdminK8sEnv := GetAndExpectStringEnvVar("SUPERVISOR_CLUSTER_KUBE_CONFIG"); svAdminK8sEnv != "" {
```

Comment:
Please keep old name KUBECONFIG instead of SUPERVISOR_CLUSTER_KUBE_CONFIG 

also make an entry in https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/docs/supervisor_cluster_setup.md